ci/build-in-docker.sh

```diff
@@ -77,7 +77,7 @@ git config --global --add safe.directory '*'
 
 bash -euxo pipefail /source/ci/install-static-desktop-file-validate.sh 0.28
 bash -euxo pipefail /source/ci/install-static-mksquashfs.sh 4.6.1
-bash -euxo pipefail /source/ci/install-static-zsyncmake.sh 0.6.2
+#bash -euxo pipefail /source/ci/install-static-zsyncmake.sh 0.6.2
```

Comment:
If it's not needed, then let's remove it altogether.

Reply:
Done, although I would not say that "it is not needed", appimagetool needs to have a working zsyncmake, the issue is that so far it hasn't work, and now the upstream website is gone.

Reply:
Removing it is a bad idea. It should rather be fixed. Please revert.

Reply:
Reverted. But I think fixing this is outside of the scope of this supposedly simple and fast PR.